fuel/converters/binarized_mnist.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+import fuel
+import h5py
+import numpy
+
+default_directory = os.path.join(fuel.config.data_path, 'binarized_mnist')
+default_save_path = os.path.join(default_directory, 'binarized_mnist.hdf5')
+
+
+def binarized_mnist(directory=None, save_path=None):
+    """Converts the binarized MNIST dataset to HDF5.
+
+    Converts the binarized MNIST dataset used in R. Salakhutdinov's DBN
+    paper [DBN] to an HDF5 dataset compatible with
+    :class:`fuel.datasets.binarized_mnist.BinarizedMNIST`.
```

Comment:
I'd be in favour of using `__init__.py` imports in `datasets/` to flatten this namespace. What do you think @bartvm?